src/getHumanAge.test.js

```diff
@@ -7,4 +7,28 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should work with zeros', () => {
+    const result = getHumanAge(0, 0);
+
+    expect(result).toEqual([0, 0]);
+  });
+
+  test('should work correct with young animals', () => {
+    const result = getHumanAge(10, 10);
+
+    expect(result).toEqual([0, 0]);
+  });
+
+  test('should work correct with older animals', () => {
+    const result = getHumanAge(15, 15);
+
+    expect(result).toEqual([1, 1]);
```

Comment:
The expected result for older animals might be incorrect. Ensure that the logic for calculating human age equivalents is correctly implemented and the expected output matches the logic.